server/routes/record-attendance/handlers/removePay.ts

```diff
@@ -0,0 +1,53 @@
+import { Request, Response } from 'express'
+import ActivitiesService from '../../../services/activitiesService'
+import PrisonService from '../../../services/prisonService'
+
+export default class RemovePayRoutes {
+  constructor(private readonly activitiesService: ActivitiesService, private readonly prisonService: PrisonService) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const { user } = res.locals
+    const { id } = req.params
+    const { attendanceId } = req.params
+
+    const instance = await this.activitiesService.getScheduledActivity(+id, user).then(i => ({
+      ...i,
+    }))
```

Comment:
Same question as in `attendanceDetails.ts`

Reply:
removed